package.json

```diff
@@ -39,11 +39,11 @@
     "prettier": "2.8.8",
     "publint": "0.1.12",
     "ts-jest": "29.1.0",
-    "turbo": "^1.9.3",
+    "turbo": "1.10.3",
     "typescript": "^5.0.0"
   },
-  "packageManager": "pnpm@8.4.0",
+  "packageManager": "pnpm@8.6.2",
   "engines": {
-    "node": ">=16.14"
+    "node": ">=18"
```

Comment:
we are still running node16 tests https://github.com/vercel/storage/actions/runs/5276021124/jobs/9542191279?pr=214

Reply:
Yes definitely 👍 this version is mostly for the developers iterating on this module, and for the CI actions that are not the unit test matrixes ones.